aptiche/lib/views/quiz/quiz_view.dart

```diff
@@ -77,24 +77,24 @@ class QuizView extends GetView<QuizController> {
                 mainAxisAlignment: MainAxisAlignment.spaceBetween,
                 crossAxisAlignment: CrossAxisAlignment.center,
                 children: <Widget>[
-                  /*  if (controller.questionIndex.value > 0)
+                  if (controller.questionIndex.value > 0)
                     CustomButton(
                       horizontalPadding: SizeConfig.safeBlockHorizontal! * 1.4,
                       verticalPadding: SizeConfig.safeBlockVertical! * 0.27,
                       text: 'Previous',
                       onTap: () {
-                        print(controller.radioGroupValue.value);
-                        //controller.previous();
+                        // print(controller.radioGroupValue.value);
+                        controller.previous();
                       },
-                    ), */
-                  CustomButton(
-                    horizontalPadding: SizeConfig.safeBlockHorizontal! * 1.4,
-                    verticalPadding: SizeConfig.safeBlockVertical! * 0.27,
-                    text: 'Clear Choice',
-                    onTap: () {
-                      controller.clearRadioGroup();
-                    },
-                  ),
+                    ),
+                  // CustomButton(
+                  //   horizontalPadding: SizeConfig.safeBlockHorizontal! * 1.4,
+                  //   verticalPadding: SizeConfig.safeBlockVertical! * 0.27,
+                  //   text: 'Clear Choice',
+                  //   onTap: () {
+                  //     controller.clearRadioGroup();
```

Comment:
I think you also reverted the comments commit

Reply:
Nope,I just removed the clear all option and in place of that added the previous button. I intentionally left this commented code if any time you want the clear all option back. Should i remove this too?

Reply:
@SriramPatibanda according to the designs we need both buttons? Or one?

Reply:
I have added switchable radio ,so if anyone want to deselect a option ,he/she can tap it once again to deselect it and previous button too need some space for a good design. So acc. to me there is no use of clear all button.